### PR TITLE
Pull upstream master

### PR DESCRIPTION
**Pull request by benbuckman**
Pull upstream master, to use token auth instead of username+password.

(Pairing with @lafave)

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.1.1 Jun 19th, 2015
```
Comment (chelsey) [inline CHANGELOG:1]: Hmm, is this your change? June 19th?

Comment (benbuckman) [inline CHANGELOG:1]: Not mine, this was the merged changelog from upstream (goshippo/shippo-node-client). They bumped to 1.1.1, we were still on 1.1.0; I made a tag on our side called `1.1.1-shyp` to differentiate.

Comment (chelsey) [inline CHANGELOG:1]: Ah, cool. good to know

**Comment by chelsey** [comment]
+1

**Comment by caseycrites** [comment]
:shaved_ice:

**Comment by caseycrites** [comment]
Let's make sure when we upgrade to this we keep the old username/password env vars out there so we can easily rollback if things break.

**Pull request merged**